forte_wrapper/spacy/spacy_processors.py

```diff
@@ -125,3 +130,20 @@ def _process(self, input_pack: DataPack):
 
         # Process sentence parses.
         self._process_parser(result.sents, input_pack)
+
+    def record(self, record_meta: Dict[str, Set[str]]):
+        r"""Method to add output type record of current processor
+        to :attr:`forte.data.data_pack.Meta.record`.
+
+        Args:
+            record_meta: the field in the datapack for type record that need to
+                fill in for consistency checking.
+        """
+        record_meta["ft.onto.base_ontology.Sentence"] = set()
+        record_meta["ft.onto.base_ontology.EntityMention"] = set()
```

Comment:
I think the correct implementation is that we need to have a `ner` flag to add `EntityMention`. But this is probably out of the scope of this PR.